scss/civicrm/event/_detail.scss

```diff
@@ -174,3 +162,22 @@
   margin-top: 20px !important;
 }
 
+
+.crm-container {
```

Comment:
why is `.crm-container` necessary?

Reply:
This specific style is for popup where `#crm-container` does not exist so, we need `.crm-container`.

Reply:
sorry @mukeshcompucorp but you need to provide more details here. Why are you talking about a popup, if the selectors don't mention any popup? Why would `#crm-container` be missing? Can you show some screenshots of what you are talking about?

Reply:
The style we are handling here is for below popup dialog.
![image](https://user-images.githubusercontent.com/26058635/36778265-47d1ef4a-1c92-11e8-977f-996a2f6e2c62.png)

Dialog is having `.crm-container` as wrapper to our form.